### PR TITLE
Bug Fix - Coefficients of reactions

### PR DESCRIPTION
**Pull request by nof21**
Fixed coefficient of reactions for the nasa9polynomials.py
Before was only read the coefficients of the reactants

### CODE REVIEW

**Review by guillemborrell — CHANGES_REQUESTED** [review]
Hi @nof21 and thanks for the patch! I've seen that the change also brings in other spurious files, like the thermopy.egg-info, and IDE-related files. Could you please remove them?

**Review by nof21 — COMMENTED** [review]
I have removed the unnecessary files.

File: thermopy/nasa9polynomials.py [hunk 1]
```diff
@@ -909,3 +909,4 @@ def __repr__(self):
         for (reag, coef) in zip(self._products, self._pcoefs):
             r = r + '+' + str(coef) + ' ' + reag.inp_name + ' '
         return """<reaction> {0}""".format(r)
+    print('test')
```
Comment (guillemborrell) [inline thermopy/nasa9polynomials.py]: Thanks! One last change request. Can you please remove this print statement too?

**Review by guillemborrell — CHANGES_REQUESTED** [review]

**Review by guillemborrell — COMMENTED** [review]
All good

**Comment by guillemborrell** [comment]
@nof21 Sorry for coming back to this after a long time. Feel free to reopen the PR so I can merge.

**Comment by nof21** [comment]
Restored and reopened

**Review by guillemborrell — APPROVED** [review]
Thanks!

**Pull request merged**